src/App.js

```diff
@@ -1,12 +1,47 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import PokemonCard from "./components/PokemonCard";
+import axios from "axios";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonListData, setPokemonListData] = useState({})
```

Comment:
Als de waarde na ophalen een array is, dan zou ik dat ook als default waarde hanteren:
```suggestion
    const [pokemonListData, setPokemonListData] = useState([])
```